Desktop/utilities/imgschemehandler.cpp

```diff
@@ -12,6 +13,8 @@ using namespace Modules;
 ImgSchemeHandler::ImgSchemeHandler(QObject *parent)
 	: QWebEngineUrlSchemeHandler{parent}
 {
+    std::cout << "ImgSchemeHandler for QtWebengine" << std::endl;
+
 	QQuickWebEngineProfile::defaultProfile()->installUrlSchemeHandler("img", this);
 }
```

Comment:
```suggestion
	std::cout << "scheme installed" << std::endl;
}
```

Reply:
same at the other one, the we know it breaks in there. But the thing is, if it does then what?

Reply:
Yes, we can do nothing when I find it‘s a Qt bug. We can only hope for future Qt solutions. Then I will leave it for now.